Rollbar/DTOs/Message.cs

```diff
@@ -1,18 +1,31 @@
 ﻿namespace Rollbar.DTOs
 {
-    using System;
-    using Newtonsoft.Json;
     using System.Collections.Generic;
     using Rollbar.Diagnostics;
 
+    /// <summary>
+    /// MOdels Rollbar Message DTO.
```

Comment:
Models, there are probably more of these

Reply:
corrected. @rokob, thanks!